system-int-test/test/cmr/system_int_test/search/collection_relevancy/collection_temporal_relevancy.clj

```diff
@@ -82,4 +83,17 @@
       [coll3 coll1 coll2 coll5 coll4]
 
       "Date range including collection with early ranges"
-      ["1955-01-01T10:00:00Z,1999-03-01T0:00:00Z"] [coll5 coll6 coll2])))
+      ["1955-01-01T10:00:00Z,1999-03-01T0:00:00Z"] [coll5 coll6 coll2])
+
+   (testing "Keyword search, sort by end date"
+     (is (d/refs-match-order? [coll3 coll4 coll1 coll2 coll5 coll6]
+                              (search/find-refs :collection {:keyword "coll"})))
+
+     (testing "Multiple ongoing collections"
```

Comment:
This `testing` block looks like it's inside the one above. Not sure if you meant to do that or not.